mono/mini/mini-arm64.c

```diff
@@ -303,9 +303,21 @@ mono_arch_cpu_optimizations (guint32 *exclude_mask)
 	return 0;
 }
 
+G_BEGIN_DECLS
```

Comment:
Why is the `G_BEGIN_DECLS` needed here?

Reply:
These are extern methods so we would need to control name mangling - although this doesn't do anything unless we're compiled as C++.

Mostly I just copied and pasted this from the Amd64 version: https://github.com/Unity-Technologies/mono/blob/unity-main/mono/mini/mini-amd64.c#L7396-L7399